### PR TITLE
Feature/display login

### PR DESCRIPTION
**Pull request by nware1066**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Testing - building tests
- [ ] Formatting - no new features
- [ ] Styling- no new features
### Detailed Description (Why is this change required? What problem does it solve? Any challenges that arose while implementing this feature? If so, how were the addressed?)

We now have a login through google that displays on the browser. We have created a file to house the api key and placed it in GitIngnore. We created a route for the login page

### Files modified:
- [ ] index.html
- [ ] App
- [ ] CardContainer
- [ ] PokeCard
- [ ] PokeForm
- [ ] Login
- [ ] FormContainer
- [ ] index.js
- [x] Other files: secrets.json

### CODE REVIEW

File: stretch-auth/src/secrets.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (AurumValian) [inline stretch-auth/src/secrets.json:1]: Nice job on the secret file!

File: stretch-auth/src/Login/Login.js [hunk 2]
```diff
@@ -1,6 +1,24 @@
...
 import './Login.css'
+import { secrets } from '../secrets';
+import { firebaseui } from '../App/App';
+import StyledFirebaseAuth from 'react-firebaseui/StyledFirebaseAuth';
+// import firebase from 'firebase';
+let firebase = require('firebase');
+console.log(secrets);
+
+
+firebase.initializeApp(secrets);
```
Comment (AurumValian) [inline stretch-auth/src/Login/Login.js:12]: There's the initialization. Good work!

**Review by AurumValian — APPROVED** [review]
Solid!

File: stretch-auth/src/secrets.json (hunk 1 above)
Comment (jordy1611) [reply stretch-auth/src/secrets.json:1]: This definitely isn't hidden/secret

**Review by jordy1611 — COMMENTED** [review]
This is totally rad
We need a new api key

File: stretch-auth/src/secrets.json (hunk 1 above)
Comment (H-Bo214) [reply stretch-auth/src/secrets.json:1]: We'll have to figure out how to truly hide this.

**Review by H-Bo214 — APPROVED** [review]

**Pull request merged**